### ISSUE TITLE
Setup user proposal guides space

### ISSUE THREAD

**Issue by fhilipk** [Project-Catalyst/catalyst#13]
Users who wish to write a proposer guide need a place to share them.

**Comment by QuasarChains** [comment]
Indeed. A general template would be useful but would need flexibility.

**Comment by fhilipk** [comment]
The idea would be its open for anyone to submit their guides.
